### ISSUE TITLE
Kopierfunktion

### ISSUE THREAD

**Issue by friedrichst** [Stephan-S/FS22_AutoDrive#801]
Hallo,

wäre es möglich eine Kopierfunktion einzufügen? Man erstellt z.B. einen Abfahrkurs und kann diesen einfach auf ein weiteres Fahrzeug kopieren, so wie es auch beim CP möglich ist.

**Comment by Tensuko** [comment]
Aber du erstellst mit AD doch keine Abfahrkurse, sondern stellst nur modus, start und ziel ein...
Sicher könnte man jetzt überlegen, ob man ein Preset dafür anlegt, aber 3 Dinge einstellen sollte doch jeder schaffen, oder ? :)

**Comment by manne70** [comment]
Ich finde das ist keie schlechte Idee, wenn man mehrere Abfahrer hat und eine lange Zielliste bei der man viel scrollen muss wäre das schon eine Erleichterung.

**Comment by friedrichst** [comment]
> Ich finde das ist keie schlechte Idee, wenn man mehrere Abfahrer hat und eine lange Zielliste bei der man viel scrollen muss wäre das schon eine Erleichterung.

Genau darum geht es mir. Manchmal sind bei mir 3-4 gleiche Abfahrer unterwegs. Und das nervt dann schon, wenn man bei jedem neu einstellen muss. Das ginge halt mit einer Kopierfunktion RuckZuck.

**Comment by Iwan1803** [comment]
Das man mit der Lupe Zielpunkte suchen kann, ist aber schon bekannt, oder?

**Comment by friedrichst** [comment]
> Das man mit der Lupe Zielpunkte suchen kann, ist aber schon bekannt, oder?

Natürlich, darum geht es aber gar nicht. Sondern darum, wenn man z.B. im Silo Weizen hat, und der Preis gerade sehr gut ist, diesen möglichst flott zu verkaufen. Und dafür setzte ich dann mal drei oder mehr Abfahrer auf einmal ein. Also muss ich bei jedem Abfahrer einzeln Abhol-, Zielpunkt und Ladung definieren. Wenn ich das nur einmal machen muss, und dann kopieren kann spart das schon viel Arbeit und Zeit.
